File: 351001/Sosnovskiy/module-publisher/src/main/java/by/egorsosnovski/distcomp/dto/TweetRequestTo.java
package by.egorsosnovski.distcomp.dto;

import jakarta.validation.constraints.Size;
import lombok.Data;

import java.util.List;

@Data
public class TweetRequestTo {
    long id;
    long creatorId;
    @Size(min = 2, max = 64)
    String title;
    @Size(min = 4, max = 2048)
    String content;
    private List<String> stickers;
}

File: 351002/Abrashin/publisher/src/main/kotlin/com/example/distcomp/dto/request/TweetRequestTo.kt
package com.example.distcomp.dto.request

import com.fasterxml.jackson.annotation.JsonRootName
import jakarta.validation.constraints.Size

@JsonRootName("tweet")
data class TweetRequestTo(
    var creatorId: Long? = null,
    @field:Size(min = 2, max = 64)
    var title: String? = null,
    @field:Size(min = 2, max = 2048)
    var content: String? = null,
    var stickers: List<String>? = null
)
